03-compose-file.md

```diff
@@ -1,78 +1,23 @@
 the expanded form.
 
-### Profiles
-
-Profiles allow to adjust the Compose application model for various usages and environments. Compose allows the user to define a set of active profiles. The exact mechanism is implementation
-specific and may include command line flags, environment variables, etc.
-
-The Services top-level element supports a `profiles` attribute to define a list of named profiles. Services without
-a `profiles` attribute set must always be enabled. A service must be ignored by the Compose
-implementation when none of the listed `profiles` match the active ones, unless the service is
-explicitly targeted by a command. In that case its `profiles` must be added to the set of active profiles.
-All other top-level elements are not affected by `profiles` and are always active.
-
-References to other services (by `links`, `extends` or shared resource syntax `service:xxx`) must not
-automatically enable a component that would otherwise have been ignored by active profiles. Instead
-Compose returns an error.
-
-#### Illustrative example
-
-```yaml
-services:
-  foo:
-    image: foo
-  bar:
-    image: bar
-    profiles:
-      - test
-  baz:
-    image: baz
-    depends_on:
-      - bar
-    profiles:
-      - test
-  zot:
-    image: zot
-    depends_on:
-      - bar
-    profiles:
-      - debug
-```
-
-- Compose application model parsed with no profile enabled only contains the `foo` service.
-- If profile `test` is enabled, model contains the services `bar` and `baz`, which are enabled by the
-  `test` profile, and service `foo`, which is always enabled.
-- If profile `debug` is enabled, model contains both `foo` and `zot` services, but not `bar` and `baz`,
-  and as such the model is invalid regarding the `depends_on` constraint of `zot`.
-- If profiles `debug` and `test` are enabled, model contains all services: `foo`, `bar`, `baz` and `zot`.
-- If Compose is executed with `bar` as explicit service to run, it and the `test` profile
-  will be active even if `test` profile is not enabled _by the user_.
-- If Compose is executed with `baz` as explicit service to run, the service `baz` and the
-  profile `test` will be active and `bar` will be pulled in by the `depends_on` constraint.
-- If Compose is executed with `zot` as explicit service to run, again the model will be
-  invalid regarding the `depends_on` constraint of `zot`, since `zot` and `bar` have no common `profiles`
-  listed.
-- If Compose is executed with `zot` as explicit service to run and profile `test` enabled,
-  profile `debug` is automatically enabled and service `bar` is pulled in as a dependency starting both
-  services `zot` and `bar`.
-
+You can also use [`include`](14-include.md) in your Compose file if you want your Compose application to be dependent on another Compose application that is managed by a different team or shared with others.
```

Comment:
Same here:
```suggestion
If you want to reuse other Compose files or factor out parts of your application model into separate Compose files, you can also use [`include`](14-include.md).
This is useful in situations where your Compose application is dependent on another application which is managed by a different team, or needs to be shared with others.
```
I think this reads better, and explains what `include` does and what it's useful for.

Reply:
Agreed! Changed 👍